fastchat/serve/gradio_block_arena_vision_anony.py

```diff
@@ -267,7 +340,7 @@ def add_text(
     if states[0] is None:
         assert states[1] is None
 
-        if len(images) > 0:
+        if len(files) > 0 and file_extension != ".pdf":
```

Comment:
This is not a reliable implementation to check whether a file is a pdf.

a file can be called "abc.pdf" but it's a jpg.

Reply:
Got it, I changed to magic number checker.